mlops/parallelm/mlops/ml_metrics_stat/classification/classification_stat_object_factory.py

```diff
@@ -0,0 +1,191 @@
+import numpy as np
+from six import string_types
+
+from parallelm.mlops.metrics_constants import ClassificationMetrics
+from parallelm.mlops.ml_metrics_stat.ml_stat_object_creator import MLStatObjectCreator
+from parallelm.mlops.mlops_exception import MLOpsStatisticsException
+
+
+class ClassificationStatObjectFactory(object):
+    """
+    Responsibility of this class is basically creating stat object for classification stat.
+    """
+
+    @staticmethod
+    def get_mlops_accuracy_score_stat_object(**kwargs):
+        """
+        Method will create MLOps Single value stat object from numeric real number - accuracy score
+        It is not recommended to access this method without understanding single value data structure that it is returning.
+        :param kwargs: numeric value of accuracy
+        :return: Single Value stat object which has accuracy score embedded inside
+        """
+        accuracy_score = kwargs.get('data', None)
+
+        single_value, category = MLStatObjectCreator. \
+            get_single_value_stat_object(name=ClassificationMetrics.ACCURACY_SCORE.value,
```

Comment:
do we need to do ".value" on each one? why not let the function perform the ".value" on its argument? code may look cleaner?